scripts/sign_bsta.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2015, Linaro Limited
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+def uuid_parse(s):
+	from uuid import UUID
```

Comment:
Please check this script with `pycodestyle` (or with `pip8.py`). There will be a lot of warnings.